src/mmw/js/src/modeling/models.js

```diff
@@ -276,10 +275,8 @@ var ScenarioModel = Backbone.Model.extend({
         this.set('inputs', new ModificationsCollection(attrs.inputs));
         this.set('modifications', new ModificationsCollection(attrs.modifications));
 
-        this.on('change:project change:name change:census', this.attemptSave, this);
-        this.get('inputs').on('add', this.attemptSave, this);
+        this.on('change:project change:name', this.attemptSave, this);
         this.get('modifications').on('add remove change', this.updateModificationHash, this);
-        this.get('modifications').on('add remove', this.attemptSave, this);
```

Comment:
If we want to keep an eye on the duration of background tasks, we can start by looking at the metrics already kept by Celery. If we want something more specific, we can use any of the following techniques to instrument the functions run by Celery:

- http://statsd.readthedocs.org/en/latest/timing.html#calling-timing-manually
- http://statsd.readthedocs.org/en/latest/timing.html#using-a-context-manager
- http://statsd.readthedocs.org/en/latest/timing.html#using-a-decorator
- http://statsd.readthedocs.org/en/latest/timing.html#using-a-timer-object-directly

The StatsD client and configuration is available across all of the machine types. Anything working through Django should already have it configured, just need to import the client:

```python
from django_statsd.clients import statsd

with statsd.timer('foo'):
    foo()
```